.github/workflows/install.yml

```
@@ -16,15 +16,15 @@ jobs:
 
     steps:
       - name: Install defang (latest)
-        run: . <(curl -Ls https://s.defang.io/install)
+        run: eval "$( curl -Ls https://s.defang.io/install)"
```

We can remove the space between `$(` and `curl`:
```suggestion
        run: eval "$(curl -Ls https://s.defang.io/install)"
```

---

oh yes, nice catch